app/models/post.rb

```diff
@@ -5,6 +5,10 @@ class Post < ApplicationRecord
 
   after_create :increment_user_posts_counter
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_inter: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] To use comment**s**_counter we should define the column in the corresponding migration 👇🏽 

```ruby
# frozen_string_literal: true

class CreatePost < ActiveRecord::Migration[7.0]
  def change
    create_table :posts do |t|
      t.string :title
      t.string :text
      t.integer :comments_counter
      t.integer :likes_counter

      t.timestamps
    end
  end
end
```